libutils/logging.c

```diff
@@ -339,6 +351,26 @@ void VLog(LogLevel level, const char *fmt, va_list ap)
     char *msg = StringVFormat(fmt, ap);
     char *hooked_msg = NULL;
 
+    if (logging_into_buffer &&
+        (level >= min_log_into_buffer_level) && (level <= min_log_into_buffer_level))
```

Comment:
```suggestion
        (level >= min_log_into_buffer_level) && (level <= max_log_into_buffer_level))
```
Are you supposed to be comparing to min and max?

Reply:
Good catch! I would swear I changed the second one to `max`.